internal/tests_config.go

```diff
@@ -54,11 +54,9 @@ type Stats struct {
 // TestsConfig represents a part of the dance configuration for tests.
 //
 // May contain prefixes; the longest prefix wins.
-//
-//nolint:govet // we don't care about alignment there
 type TestsConfig struct {
 	Default status   `yaml:"default"`
-	Stats   Stats    `yaml:"stats"`
+	Stats   *Stats   `yaml:"stats"`
```

Comment:
What do mean stats? Let's add a comment.

Reply:
The `Stats` struct is defined above it. Do you prefer to add a comment above the `Stats` struct, or next to this field?